components/modal/modals/ServerIdentity.tsx

```diff
@@ -0,0 +1,345 @@
+import { BiRegularX, BiSolidEditAlt, BiSolidSave } from "solid-icons/bi";
+import { Show, createSignal } from "solid-js";
+
+import { ServerMember } from "revolt.js";
+
+import { clientController } from "@revolt/client";
+import { useTranslation } from "@revolt/i18n";
+import { userInformation } from "@revolt/markdown/users";
+import {
+  Avatar,
+  Button,
+  CategoryButton,
+  Column,
+  Input,
+  NonBreakingText,
+  OverflowingText,
+  Preloader,
+  Row,
+  Typography,
+  Username,
+  styled,
+  useTheme,
+} from "@revolt/ui";
+import { generateTypographyCSS } from "@revolt/ui/components/design/atoms/display/Typography";
+
+import { PropGenerator } from "../types";
+
+const [avatarId, setAvatarId] = createSignal(null);
+const [uploading, setUploading] = createSignal<boolean>();
+
+async function uploadToAutumn(file) {
```

Comment:
You should use `File | Blob` instead of not providing a type (aka. implicit `any`) since the `FileList` type is just an array of `File`s (See MDN's article on FileList: https://developer.mozilla.org/en-US/docs/Web/API/FileList)
```suggestion
async function uploadToAutumn(file) {
```